scripts/start.sh

```diff
@@ -155,6 +155,9 @@ OPTS+=" $jvm_args -Djava.library.path=$JAVA_LIB_PATH"
 OPTS+=" -Dhttp.port=$port"
 OPTS+=" -Ddb.default.url=$db_loc -Ddb.default.user=$db_user -Ddb.default.password=$db_password"
 
+# set Java related options (e.g. -Xms1024m -Xmx1024m)
+export JAVA_OPTS="-XX:+HeapDumpOnOutOfMemoryError"
```

Comment:
why not use  JAVA_OPTS+= ?
we usually set these variables as part of our initd script